### DISCUSSION TITLE
Run SimpleLogin Docker Containers

### DISCUSSION THREAD

**Discussion by connor1mcdonald** [simple-login/app#441] [Q&A]
Hi all,

Not sure if I'm being stupid but I'm not sure of the steps in the screenshot.

Anyone able to advise on what I should do with "~/simplelogin.env" and "FLASK_SECRET"?

Thanks,
C

![image](https://user-images.githubusercontent.com/47890978/115997564-68d2d300-a5db-11eb-89ad-cde24353ae83.png)

**Comment by nguyenkims** [comment]
Hi `~/simplelogin.env` is a file at your `$HOME` location on your server. The `FLASK_SECRET` is mentioned in the config file 

![2021-04-26 at 07 28](https://user-images.githubusercontent.com/3167780/116033068-0b816500-a661-11eb-8513-52168dc6a2fd.png)
